intermediate-exercises/src/components/01-LifeCycle.js

```diff
@@ -20,6 +21,8 @@ class LifeCycle extends React.Component {
     // Task 1: Display a message "I'm mounted!" in developer's console when the
     //         component finishes mounting.
     //         Use `console.log` function for it.
+    // eslint-disable-next-line no-console
+    console.log('I\'m mounted!');
```

Comment:
You can use the template literals here instead of disabling the eslint.

Reply:
I used `no-console` disable rule. As I need to use console.log here, so I had to disable eslint here.